content/articles/statistical-hypothesis-testing-python-implementation/index.md

```diff
@@ -0,0 +1,488 @@
+---
+layout: engineering-education
+status: publish
+published: true 
+url: /statistical-hypothesis-testing-python-implementation/
+title: Statistical Hypothesis Testing With Python Implementation
+description: This tutorial is a comprehensive explanation and implementation of statistical hypothesis testing methods such as t-test, chi-squared test & ANOVA in python, It is well known that most of the implementation of these techniques are always in R, and most Data Scientists and Analysts who use python always shy away from it.
+author: qoyum-olatunde-yusuf
+date: 2021-06-16T00:00:00-11:00
+topics: [Statistical Analysis, Hypothesis Testing, Data Science, Python ]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/statistical-hypothesis-testing-python-implementation/hero.png
+ alt: statistic hypothesis testing with python cover image
+ 
+---
+Hypothesis is the act of making observations and generating probing questions from the observations made in an attempt to come to a conclusion. Hypothesis testing is the test of assumptions on a population sample. It involves checking out whether a hypothesis should be accepted or not. In the world of work, hypothesis testing has been of great importance in making worthy decisions off mere observations. Billions of data are being generated on a daily basis around the globe and this has made the relevance of hypothesis testing a stronghold in making business decisions.
+<!--more-->
+
+### Understanding Hypothesis Testing
+
+There are two types of hypothesis testing, namely:
+1. Null Hypothesis
+2. Alternate Hypothesis
+
+**Null hypothesis** is the initial assumption about an event (also referred to as ground truth), whereas alternate hypothesis is an assumption that counters the initial assumption"
+
+In a scenario whereby a septuagenarian woman has a swollen stomach or baby bump, our initial assumption is going to be that 
+**"She is pregnant"**.
+
+while our alternate hypothesis will be
+**"She is not pregnant"**.
+
+The null hypothesis is a statement believed to be true unless we can prove otherwise beyond a reasonable doubt.
+
+In trying to assess two [mutually exclusive](https://www.mathsisfun.com/data/probability-events-mutually-exclusive.html) facts on a certain sample of data. We will refer to H0 as our null hypothesis and H1 as our alternate hypothesis, there seem to be too many alternate names for H0(ground truth, initial assumption, prior assumption, e.t.c) whereas H1 is vaguely described.
```

Comment:
This does not make any sense at all.

Reply:
I need to point this out, i think it is rude using the "does not make sense at all" sentence, this is the second time you will be using it, it is almost the same as saying the writer is not intelligent at all. I think there are better way to go about reviews and correct mistakes politely, this is not my first tutorial here. It is highly important we respect one another.

Reply:
you're not making any point with it, in the first review, you told me to add additional resources to the "mutually exclusive" which i believe was pointed out because of people that might not be familiar with the concept. and that's the only thing I've added to the paragraph. If we are assumed to be perfect, the tutorials wont even go through reviews at all, but we also learn more when we write about what we've learnt, so please help me help myself more.
Thanks

Reply:
I apologize, I didn't know you'd take it that way.
The reason why I said so was because you have copied what I've told you as a comment and blatantly pasted it. I'm sure you can cross-check it. If you take a second look at it, you can see that the reader cannot interpret that statement.
Once again, my apologies, and thanks.

Reply:
Also, as I've mentioned before, I'd like you to take a look at your peer's article drafts, which I remember mentioning twice as well. The subsection `Three-Way ANOVA` is in H2 whereas the rest are in H1. At the same time, if ANOVA is the main topic and one-way, two-way and three-way ANOVA are the subtopics, the subtopics would be in H4 and not H3. These small things can easily be picked up by going through the standard template that I had sent you in the first review or, like I've said in the past two reviews, do take a look at other article drafts so that you will get an idea on the formatting and little things like that.
I understand where you're coming from and I sincerely apologize for you feeling that way, but these things can easily be picked up from your end as well.
My apologies, and thanks.

Reply:
Thanks a bunch, I am sorry to have caused so much trouble for you, its a large oversight on my part which shouldn't have happened. I apologize for that.  
I will pay more attention to my formatting going forward.